libs/notification/src/tokens/notification-clicks.ts

```diff
@@ -0,0 +1,14 @@
+import {inject, InjectFlags, InjectionToken} from '@angular/core';
+import {SwPush} from '@angular/service-worker';
```

Comment:
Now if somebody just wants client side notifications they have to add service worker package?

Reply:
✅

See new utility `provideSwPush`